### ISSUE TITLE
Don't work with Gitlab groups

### ISSUE THREAD

**Issue by bibendi** [rscherf/GitLink#34]
Hi!
The link generation doesn't work well with Gitlab groups.
For example, `git@gitlab.selfhost.io:group1/group2/repo-name.git` => `https://gitlab.selfhost.io/group1/group2/-/blob/master/some-file.md`. But it should be like `https://gitlab.selfhost.io/group1/group2/repo-name/-/blob/master/some-file.md`

**Comment by kayoo123** [comment]
same error to me...

**Comment by rscherf** [comment]
Feel free to submit a PR ensuring backwards compatibility.

**Comment by kayoo123** [comment]
I'm very bad with python; I think it's where you get `repo` variable...
(I see, we use custom private domain gitlab, maybe it's HOSTINGS value which are not compatible)

I read that you use : `git rev-parse ...`, but it show empty when I try with cli.

Have you think of another way to get this info ? 

maybe like this : 
```bash
## Get remote URL
git config --get remote.origin.url
    git@gitlab.jeremi.fr:group/project.git

## Translate to HTTPS
git config --get remote.origin.url |sed -r 's/git@([^:]*):(.*).git/https:\/\/\1\/\2/'
    https://gitlab.jeremi.fr/group/project
```

Hope this help...

**Comment by kayoo123** [comment]
no news ?

**Comment by kayoo123** [comment]
As I don't understand why I'm being downvoted, I'm sharing an alternative version using the build system 

here's the link: https://github.com/kayoo123/sublimeTextBuildSystem/tree/main?tab=readme-ov-file#giturltoclipboard

**Comment by michaelblyons** [comment]
Downvote for assuming updates. Maintainer has already stated he'd consider a PR if it didn't cause regressions. I didn't downvote any of your other comments.

You're unwilling to write any Python. Would you be interested in experimenting with just a regex change?

https://github.com/rscherf/GitLink/blob/master/GitLink.py#L82

**Comment by ddelange** [comment]
right, I just ported #36 to #37 (I'm not a maintainer and I always run git over HTTPS). @eread do you have time to do a one line PR?
